pypsa/statistics.py

```diff
@@ -248,60 +248,6 @@ def filter_bus_carrier(n, c, port, bus_carrier, df):
         )
 
 
-def aggregate_components(
```

Comment:
any way to make this backwards compatible in case someone used this as imported function

```py
from pypsa.statistics import aggregate_components
```

Reply:
checked pypsa-eur and pypsa-earth to see if other people are using this function but did not find any use cases. Since this functions seems like only being used for statistics, I changed it to a private method